Assignment-1

msmacbeth opened the pull request:
Completed Assignment 1 

- Created .env file to store paths.
- Loaded variables using dotenv 
- Loaded/located parquet files using glob (somehow I made an error in the path that duplicated the name, which created subfolders as a result so at this point I learned about recursive search to locate these files - that was a fun, time consuming detour. 
- Converted Dask DF to Pandas DF for easier calculation and manipulations
- Added new columns for analysis: Lag, Returns, HI-Lo range 
- Calculate rolling 10 day avg

ananyajha2000 commented:
Please only commit required files and silence warnings, makes it very hard to load
A1 incomplete- need to use group_by ticker for features and rolling avg

msmacbeth-git committed:
remove unnecessary files and update .gitignore

msmacbeth-git committed:
remove unnecessary files from tracking

msmacbeth commented:
Hi Ananya, 

I've cleaned up the files, it'll be MUCH easier now!  I've updated to add groupby ( I didn't see that in the file so I didn't realize it was a requirement), it's looking good now.

I noticed that in the raw file it lists all the parquet files, but in VS code it does not.  Is that normal?   

I added 05_src/process_data.py files to .gitignore and all but 1 have successfully been 'ignored'.  Can you help me understand why that last one remains?  Not sure why that one didn't get hidden.

msmacbeth-git committed:
Assignment-1 Finalized

msmacbeth-git committed:
Updated .gitignore to exclude unnecessary files

msmacbeth-git committed:
Clean up unnecessary files from PR

msmacbeth-git committed:
Clean up files from PR

msmacbeth-git committed:
Cleaning up PR based on updated .gitignore

msmacbeth-git committed:
Adding assignment_1.ipynb

msmacbeth-git committed:
Update .gitignore to exclude unnecessary files

msmacbeth-git committed:
Clean up PR files

msmacbeth-git committed:
Remove unnecassary files from tracking and update .gitignore

msmacbeth-git committed:
Resolved conflicts, keeping essential files only

msmacbeth-git committed:
Updated to exclude unnecessary files

msmacbeth-git committed:
clean up files

msmacbeth-git committed:
Cleaning the PR files

msmacbeth commented:
PR has been cleaned of 11k+ files.  However, I'm unable to reduce beyond 135 files.  

.gitignore file has been updated extensively
Assignment-1 has been updated and now includes 'groupby'. 

Reached out to Ananya for assistance to reduce PR files from 135.  Otherwise, the assignment is ready for review.

ananyajha2000 commented:
A1 complete!
